### PR TITLE
Geofenceapi.org integration

### PR DESCRIPTION
**Pull request by blockjon**
Here's my first crack at integrating with GeofenceAPI.org.

The idea here is that someone who's creating a GeoJSON document can import their work directly into GeofenceAPI.org in just one step.

geojsonio

I have intentionally only done a small amount of work here to demonstrate the integration and solicit feedback.

@tmcw in reference to https://github.com/mapbox/geojson.io/issues/526

### CODE REVIEW

File: src/source/geofenceapi.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+var fs = require('fs');
```
Comment (tmcw) [inline src/source/geofenceapi.js]: looks like this and the `tmpl` variable aren't used

**Comment by tmcw** [comment]
besides the inline comment about unused vars, this looks pretty good!

**Comment by blockjon** [comment]
@tmcw Per your comment, I just removed what i think are more unused lines via https://github.com/mapbox/geojson.io/pull/527/commits/3be77a9e08587cd7c7b0ce4d06856d116caf7a66

On my end, as I move forward, I will "harden" the landing page I created today unit & integration tests and also streamline the user's experience one they land.

At this stage, I will commit to being extremely careful to keep it working and available via my chatroom tool and/or github if someone reports a glitch.

What else can I provide to you in advance of us merging & deploying?

**Comment by blockjon** [comment]
It's ready @tmcw

**Comment by tmcw** [comment]
👍 merging, thanks!

**Comment by blockjon** [comment]
🎉

**Comment by blockjon** [comment]
@tmcw I'm shutting the service down since it hasn't proven viable to keep running. I tried to create a revert but GH prevented it. Let's knock the link out for Geofenceapi.

**Pull request merged**